step-definitions/Hotels/Sprint2SD.js

```diff
@@ -0,0 +1,264 @@
+})
+
+
+
+
+
+// TC17
+
+When(/^I click on Dates$/, async function() {
+    await hotelsHome.clickDates();
+    await browser.pause(3000);       
+})
+
+When(/^I go to current month$/, async function() {
+    await hotelsHome.goToMonthBackwards();
+    await browser.pause(3000);       
+})
+
+Then(/^I verify Past dates -if any- are disabled$/, async function () {
+    expect(await hotelsHome.september2022FromTodayEnabled(-2), 'Past dates are enabled').to.be.false;
+})
+
+Then(/^I verify Back button on current month is disabled$/, async function () {
+    expect(await hotelsHome.backMonthButtonDisabled(), 'Back button is enabled').to.be.false;
+})
+
+
+
+
+
+// TC29
+
+When(/^I click on List your property$/, async function() {
+    await hotelsHome.clickProperty();
+    await browser.pause(3000);       
+})
+
+Then(/^I verify What would you like to list is displayed$/, async function () {
+    await hotelsHome.switchToListTab()
+    expect(await hotelsHome.wouldYouLikeDisplayed(), 'Not displayed').to.be.true;
+})
+
+Then(/^I verify Lodging and Private residence options are displayed$/, async function () {
+    expect(await hotelsHome.isPrivateDisplayed() && await hotelsHome.isLodgingDisplayed(), 'Not displayed').to.be.true;
+})
+
+When(/^I click on Private residence$/, async function() {
+    await hotelsHome.clickPrivate();
+    await browser.pause(3000);       
+})
+
+Then(/^I verify Step 1 of 3 is displayed$/, async function () {
+    expect(await hotelsHome.isStep1Displayed(), 'Not displayed').to.be.true;
+})
+
+Then(/^I verify See how much you could earn! is displayed$/, async function () {
+    expect(await hotelsHome.isEarnDisplayed(), 'Not displayed').to.be.true;
+})
+
+When(/^I enter 4 bedroom$/, async function() {
+    await hotelsHome.clickBedroom();
+    await hotelsHome.clickBedroom();
+    await hotelsHome.clickBedroom();
+    await hotelsHome.clickBedroom();
+    await browser.pause(3000);       
+})
+
+When(/^I enter 2.5 bathroom$/, async function() {
+    await hotelsHome.clickBathroom();
+    await hotelsHome.clickBathroom();
+    await hotelsHome.clickBathroom();
+    await browser.pause(3000);       
+})
+
+When(/^I click on Next button$/, async function() {
+    await hotelsHome.clickNext();
+    await browser.pause(3000);       
+})
+Then(/^I verify Step 2 of 3 is displayed$/, async function () {
+    expect(await hotelsHome.isStep2Displayed(), 'Not displayed').to.be.true;
+})
+
+Then(/^I verify Where is your property located is displayed$/, async function () {
+    expect(await hotelsHome.isPropertyDisplayed(), 'Not displayed').to.be.true;
+})
+
+When(/^I enter 121 in address$/, async function() {
+    await hotelsHome.enterAddressText('1211 6th Ave');
+    await browser.pause(3000);       
+})
+
+When(/^I select 1211 6th Avenue, New York, NY, USA from auto-suggestion$/, async function() {
+    await hotelsHome.intoGoingToAddress('1211 6th Avenue, New York, NY, USA');
```

Comment:
Any data like date/address/Phone number/ num of room etc should be passed dynamically from Feature file as parameter. This comment applicable for any steps-definition that passing data.